src/Maker/MakeEntity.php

```diff
@@ -376,6 +382,66 @@ private function askForNextField(ConsoleStyle $io, array $fields, string $entity
             $data['nullable'] = true;
         }
 
+        $data['apiFilter'] = null;
+
+        if (true === $apiOptions) {
+            $availableFilters = [
+                'SearchFilter',
+                'DateFilter',
+                'BooleanFilter',
+                'NumericFilter',
+                'RangeFilter',
+                'ExistsFilter',
+                'OrderFilter',
+                'MatchFilter',
+                'TermFilter',
+            ];
+
+            $apiFilter = null;
+            while (null === $apiFilter) {
+                $question = new Question('Do you want to add a filter for your api resource? (enter <comment>?</comment> to see all filters)', 'No');
+                $question->setAutocompleterValues($availableFilters);
```

Comment:
This list should be filtered to include only filters compatible with the type of the current property.

Reply:
Done. But i also added features for managing the short/full className because autocompletion, we probably need to display shortClassName but we want works with FQCN.